doc/source/installation.rst

```diff
@@ -0,0 +1,66 @@
+Installation
+============
+
+Supported systems and versions
+------------------------------
+
+.. image:: https://img.shields.io/travis/quantling/pyndl/master.svg?maxAge=3600&label=Linux
+    :target: https://travis-ci.org/quantling/pyndl?branch=master
+
+.. image:: https://img.shields.io/pypi/pyversions/pyndl.svg
+    :target: https://pypi.python.org/pypi/pyndl/
+
+*pyndl* currently only supports installation on Linux systems. However, it is
+possible to install it on other operating systems, but be aware
+that some functionality might not work or will not work as intended.
+
+.. note::
+
+  We recommend to install `Minicoda <https://conda.io/miniconda.html>`_ before
+  installing *pyndl* or to create a virtualenv within your personal folder.
+
+
+Linux
+-----
+
+If you want to install *pyndl* on Linux the easiest way is to install it
+from `pypi <https://pypi.python.org/pypi>`_ with:
+
+.. code:: bash
+
+    pip install pyndl --user
+
+MacOS
+-----
+
+If you want to install *pyndl* on MacOS you can also install it from
```

Comment:
xcode is not preinstalled on macOS system. Therefore first instruction may refer to an installation instruction.

Reply:
Thank you, I added this information in [055abfe](https://github.com/quantling/pyndl/pull/112/commits/055abfee9236836e7be8ea7d0eaea9ec29a9c0bf)